src/workspace.ts

```diff
@@ -95,18 +95,32 @@ export async function patchSettings(patch: any, askText: string): Promise<boolea
 
     // Patch has been applied in-memory, and now we have to update the file (settings.json).
     // Ask for permission.
-    let allow = await presenter.askYesNo(askText);
+    let allow = await askForPermission();
     if (!allow) {
         return false;
     }
 
     let content = JSON.stringify(settings, null, 4);
     fs.writeFileSync(filePath, content);
-    Feedback.info("Updated settings.json.");
+    Feedback.info(`Updated ${filePath}.`);
 
     return true;
 }
 
+export async function askPatchSettings(patch: any, filePath: string, askText: string): Promise<boolean> {
+    async function askForPermission(): Promise<boolean> {
+        return await presenter.askYesNo(askText);
+    }
+    return await patchSettingsOnConfirm(patch, filePath, askForPermission);
+}
+
+export async function patchSettings(patch: any, filePath: string): Promise<boolean> {
```

Comment:
If the function above is renamed, then this one can be renamed as well: `patchSettingsWithoutPrompt`?

Reply:
Done.